security/ndn-lite-sec-config.c

```diff
@@ -37,11 +37,10 @@ ndn_security_init(void)
   // HMAC backend
   ndn_lite_default_hmac_load_backend();
 
+  // RNG fake backend
+  ndn_lite_default_rng_load_backend();
+
   if (platform_security_init != NULL) {
     platform_security_init();
   }
-  ndn_rng_backend_t* rng_backend = ndn_rng_get_backend();
```

Comment:
Where is the RNG supplied to uECC? It should be either here or before every invocation of `uECC_sign` and `uECC_make_key`.

Reply:
Yes it's still before every invocation of `uECC_sign` and `uECC_make_key`, in the form of loading default backend, like other security features above.